src/app/_css/globals.css

```diff
@@ -25,103 +27,121 @@
   --dark-rock-800: #403F4C;
   --soft-white-100: #F6F6F6;
 
-    /* Typography */
+  /* Typography */
   --colfax-medium: "Colfax Medium", Roboto, sans-serif;
-  --colfax-regular: Colfax, Roboto, sans-serif;
-  /* TODO: Replace placeholder TNR for actual Acta*/
+  --colfax-regular: "Colfax Regular", Roboto, sans-serif;
   --acta-book: "Acta Book", "Times New Roman", serif;
+  --acta-bold: "Acta Bold", "Times New Roman", serif;
+
+  /* Mobile font-sizes */
+  --mobile-subtitle-01: 16px;
+  --mobile-subtitle-02: 14px;
+  --mobile-body-01: 16px;
+  --mobile-body-02: 14px;
+  --mobile-overline: 12px;
+  --mobile-caption: 10px;
+
+  /* Desktop font-sizes*/
+  --desktop-subtitle: 28px;
+  --desktop-body: 20px;
+  --desktop-caption: 16px;
+  --desktop-small: 14px;
+  --desktop-xsmall: 12px;
+  --mobile-small: 10px;
+}
 
-  *, *::before, *::after {
-    box-sizing: border-box;
-  }
+*, *::before, *::after {
+  box-sizing: border-box;
+}
 
-  * {
-    margin: 0;
-  }
+* {
+  margin: 0;
+}
 
-  body {
-    font-size: 14px;
-    text-rendering: optimizeLegibility;
-    line-height: 1.5;
-    -webkit-font-smoothing: antialiased;
-  }
+body {
+  font-size: 14px;
+  text-rendering: optimizeLegibility;
+  line-height: 1.5;
+  -webkit-font-smoothing: antialiased;
+  font-family: var(--colfax-regular);
+  font-size: var(--mobile-body-01)
+}
 
-  img, picture, video, canvas, svg {
-    display: block;
-    max-width: 100%;
-  }
+img, picture, video, canvas, svg {
+  display: block;
+  max-width: 100%;
+}
 
 
-  input, button, textarea, select {
-    font: inherit;
-  }
+input, button, textarea, select {
+  font: inherit;
+}
 
-  p, h1, h2, h3, h4, h5, h6 {
-    overflow-wrap: break-word;
-  }
+p, h1, h2, h3, h4, h5, h6 {
+  overflow-wrap: break-word;
+}
 
-  #root, #__next {
-    isolation: isolate;
-  }
+#root, #__next {
+  isolation: isolate;
+}
+
+a {
+  text-decoration: none;
+  color: var(--dark-rock-800);
+}
+
+h1 {
+  font-size: 56px;
```

Comment:
Fun fact, it looks to me (though I haven't looked deeply into Figma) that this is using a base 4 system, that uses multiples of 4, with the exception of 10px and 14px. Sometimes we define spacings/sizes as variables, like `--font-size-4: 16px`, or `--space-5: 20px` (5 * 4 = 20). Tailwind uses this type of naming for example

Reply:
so instead of changing h1, ... , h5 it would be better to declare sizings / spacings as variables?

Reply:
I think it makes sense to declare the font size variables, and then apply the styles here like you did with
```
h1 {
  font-size: var(--wtv)
}
```
But with the variables, if you need to make a `<p>` 56px somewhere, you can still use the variable. However, doing this for spacings requires you to be fairly certain of all the spacings the design uses (which sometimes is tricky), so there's more work upfront, and it may be overkill, depending on the project. I'll leave it up to you :)

Reply:
got it, updated accordingly. 
as i'm picking up patterns in spacings the `--space` vars for the margins / gaps should appear eventually, makes sense?